build/squire-raw.js

```diff
@@ -2254,17 +2254,33 @@ var onDrop = function ( event ) {
     while ( l-- ) {
         switch ( types[l] ) {
         case 'text/plain':
+        case 'Text': // IE Specific
             hasPlain = true;
             break;
         case 'text/html':
             hasHTML = true;
             break;
         default:
-            return;
+            break;
         }
```

Comment:
Safari will include a bunch of mac specific types as well as `text/plain`. In Chrome, if you drop a hyperlink it will include a `uri-list` as one of the types, and firefox includes `x-moz-url`.